test/system/fssSystemTest.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
`fssSystemTest.py -> fss_system_test.py`

Reply:
This goes against everything I believe in, but, okay <sigh> ... haha

Reply:
https://www.python.org/dev/peps/pep-0008/#package-and-module-name 😉